<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryWidget</class>
 <widget class="QWidget" name="InventoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#InventoryWidget {
    background-color: #f5f5f5;
}

QTableView#InventoryTable {
    background-color: #ffffff;
    border: 1px solid #dcdcdc;
    border-radius: 5px;
}

QPushButton#LoadButton,
QPushButton#AddProductButton,
QPushButton#DeleteProductButton {
    font-size: 12px;
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 5px;
    cursor: pointer;
    transition: background-color 0.3s;
}

QPushButton#LoadButton:hover,
QPushButton#AddProductButton:hover,
QPushButton#DeleteProductButton:hover {
    background-color: #2980b9;
}
</string>
  </property>
  <widget class="QTableView" name="InventoryTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddProductButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProductIDLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Product ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmountLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProductIDLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AmountLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ProductIDLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Product ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProductIDLineEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteProductButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Product</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
